### PR TITLE
Styling and Linters

### PR DESCRIPTION
**Pull request by David-Verde**
Add SCSS Style
Install SASS dependencies on project
Fix linter using Eslint and Stylelint Extentions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "eslint-plugin-jsx-a11y": "^6.6.0",
     "eslint-plugin-react": "^7.30.1",
```
Comment (MeqdamAlqudah) [inline package.json:55]: - [ ] Please there is an issue in the code 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @David-Verde , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.jsx [hunk 2]
```diff
@@ -76,11 +75,11 @@ const BookForm = () => {
...
-        <div className="col-md-2 pb-5 pb-md-0">
-          <button type="submit" onClick={addBookToList}>
-            ADD BOOK
-          </button>
-        </div>
+
+        <button type="submit" className="add-book-btn btn" onClick={addBookToList}>
+          ADD BOOK
+        </button>
+
```
Comment (ShahierNashaat) [inline src/components/BookForm.jsx:82]: - [x] Kindly be noticed that adding a book is not working please fix that.
- [x] Also the `ADD BOOK` button goes over the categories select menu please fix that.

Screenshot from 2022-07-15 00-09-29

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @David-Verde , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.jsx (hunk 2 above)
Comment (David-Verde) [reply src/components/BookForm.jsx:82]: ok i fix both Issues thanks for let me know.

**Review by Deepakdanger — APPROVED** [review]
Hi @David-Verde, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**